participant recruitment (#17)
katiedrax/rr_funding: protocol/protocol.Rmd

Our aim is that people will be sampled to achieve an equal distribution across the cells shown in the Table 1, however this may be affected by availability. 

> We will identify prospective participants using publicly available information and personal contacts within the respective partnerships.

Reviewer: ```suggestion
We will identify prospective participants using publicly available information and contacts with personnel from the journals and funders involved in partnerships. 
```

> Where possible, we will establish contact with individuals by email to provide them with information about the study and invite them to participate. 

Reviewer: ```suggestion
```

> If they choose to participate we will organise a mutually convenient time to conduct the interview. 

Reviewer: ```suggestion
```
